other/final/no-console-5.js

```diff
@@ -85,6 +51,53 @@ module.exports = {
         },
       })
     }
+
+    function reportDeniedMethodUsage(identifier) {
+      if (isDisallowedFunctionCall(identifier)) {
+        context.report({
+          node: identifier.parent.property,
+          message: 'Using console is not allowed',
+        })
+      } else {
+        const variableDeclaratorParent = findParent(
+          identifier,
+          parent => parent.type === 'VariableDeclarator',
+        )
+        if (variableDeclaratorParent) {
+          const references = context
+            .getDeclaredVariables(variableDeclaratorParent)[0]
+            .references.slice(1)
+
+          references.forEach(reference => {
+            if (
+              findParent(
+                reference.identifier,
+                parent => parent.type === 'VariableDeclarator',
+              )
+            ) {
+              return reportDeniedMethodUsage(reference.identifier)
```

Comment:
I have a rule enabled called `consistent-return` which requires that if you return something anywhere you have to return something everywhere. Makes things more consistent. In this case we don't actually care about the return value, but the code makes it look like the return value does matter. So to fix this, just put the return statement on the next line by itself :smile: